### PR TITLE
[TAS-531] 🎉 Init 'add-iscn-qr-code-to-epub'

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

File: add-iscn-qr-code-to-epub/index.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+/**
+ * @param {CheerioAPI} opf$ $ of content.opf
+ * @param {string} iscnPageHref path to ISCN page
+ * @param {string} iscnQRCodeHref path to ISCN QR code image
+ */
+function updateContentOPF(opf$, iscnPageHref, iscnQRCodeHref) {
+  const manifest = opf$('manifest');
+  manifest.append(`  <item id="a" href="iscn.css" media-type="text/css" />\n  `);
+  manifest.append(`  <item id="b" href="${iscnQRCodeHref}" media-type="image/png" />\n  `);
```
[the remark this answers is not in the document]
Comment (WeiJun0827) [reply add-iscn-qr-code-to-epub/index.js:111]: Oh, my bad. Forgot to update after testing POC

File: add-iscn-qr-code-to-epub/index.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    setISCNLink(iscnXHTML$, iscnPrefix);
+    addDepubDisclaimer(iscnXHTML$);
+    const updatedISCNXHTMLString = iscnXHTML$.xml();
+    fs.writeFileSync(iscnXHTMLPath, updatedISCNXHTMLString, 'utf-8');
+
+    // zip
+    const epubFilename = path.basename(epubPath);
+    const zip = new AdmZip();
+    zip.addLocalFolder(unzippedFolderPath);
+    zip.writeZip(`${outputFolder}/${epubFilename}`);
```
[the remark this answers is not in the document]
Comment (WeiJun0827) [reply add-iscn-qr-code-to-epub/index.js]: How about using the `archiver` lib as [epub-gen does](https://github.com/cyrilis/epub-gen/blob/master/lib/index.coffee#L345-L360).

**Review by williamchong — APPROVED** [review]
needing both `adm-zip` and `archiver` seems weird, I think `jszip` can handle both task

**Comment by WeiJun0827** [comment]
> needing both `adm-zip` and `archiver` seems weird, I think `jszip` can handle both task

Well, that's true. 
I found that `jszip`'s APIs are more like asynchronous reading zip files into memory and should loop through each one and manually save with `fs.writeFile` then.
But `adm-zip` can unzip an epub to a folder with just one line `extractAllTo`, though it seems it does not support compressing with a particular compression level.
Is `jszip` still preferred in our case?

**Comment by nwingt** [comment]
# Updated page style 

### Before
Screenshot 2023-11-01 at 13 31 45

### After
Screenshot 2023-11-01 at 13 29 06

**Pull request merged**